scripts/setup_linux.sh

```diff
@@ -219,7 +219,7 @@ conda activate garage
   # See https://github.com/openai/gym/issues/100
   # See https://github.com/pybox2d/pybox2d/issues/82
   pip uninstall -y Box2D Box2D-kengz box2d-py
-  pip install Box2D
+  pip install Box2D box2d-py
```

Comment:
i think you need to pick one, because these have the same python namespace.

See the issue links above for info on the dumpster going on with gym+Box2D right now.

We've implemented a couple fixes for this across the install scripts/Dockerfile. Can you verify which fix actually works and make sure they all use the same fix?

Reply:
OK I will check